data/publiccloud/terraform/azure.tf

```diff
@@ -35,6 +35,14 @@ variable "image_id" {
     default = ""
 }
 
+variable "image_uri" {
+	default = ""
+}
+
+variable "publisher" {
```

Comment:
why do we need this variable ? I have doubts that this file will work with anything except SUSE publisher without further tweaking ...

Reply:
Because the uploaded images via the Compute Gallery are published via the `qe-c` publisher by default, to avoid confusion with existing SUSE images.

Reply:
Can I close this?

Reply:
ah ok I see , yes now it make sense